### PR TITLE
Add support for left-handed users

### PR DESCRIPTION
**Pull request by arturitu**

### CODE REVIEW

File: src/orientation-arm-model.js [hunk 1]
```diff
@@ -78,8 +80,12 @@ export default class OrientationArmModel {
   }
 
   setLeftHanded(isLeftHanded) {
-    // TODO(smus): Implement me!
     this.isLeftHanded = isLeftHanded;
+    if(isLeftHanded){
```
Comment (cvan) [inline src/orientation-arm-model.js]: probably want to add spaces: `if (isLeftHanded) {`

File: src/orientation-arm-model.js [hunk 2]
```diff
@@ -13,7 +13,9 @@
  * limitations under the License.
  */
 
-const HEAD_ELBOW_OFFSET = new THREE.Vector3(0.155, -0.465, -0.15);
+const HEAD_ELBOW_OFFSET_RIGHTHANDED = new THREE.Vector3(0.155, -0.465, -0.15);
+const HEAD_ELBOW_OFFSET_LEFTHANDED = new THREE.Vector3(-0.155, -0.465, -0.15);
+let head_elbow_offset = HEAD_ELBOW_OFFSET_RIGHTHANDED;
```
Comment (cvan) [inline src/orientation-arm-model.js]: i'd change this to be `camelBack`-cased (i.e., `headElbowOffset`) like the other local variables

File: src/ray-controller.js [hunk 3]
```diff
@@ -59,7 +59,8 @@ export default class RayController extends EventEmitter {
     this.isTouchActive = false;
     // Is this a synthetic mouse event?
     this.isSyntheticMouseEvent = false;
-
+    //  Is lefthanded
```
Comment (cvan) [inline src/ray-controller.js]: I'd remove the leading whitespace and add puncutation:

```js
// Is the user left handed?
```

File: src/ray-controller.js [hunk 4]
```diff
@@ -79,7 +80,15 @@ export default class RayController extends EventEmitter {
 
     var gamepad = this.getVRGamepad_();
 
+    this.gamepad = gamepad;
     if (gamepad) {
+      if(gamepad.hand){
+        if(gamepad.hand === 'left'){
```
Comment (cvan) [inline src/ray-controller.js]: I'd replace these couple lines with this:

```js
this.isLeftHanded = gamepad.hand === 'left';
```

**Comment by cvan** [comment]
nice work! :fist_left:

File: src/orientation-arm-model.js (hunk 2 above)
Comment (borismus) [reply src/orientation-arm-model.js]: Agree

File: src/orientation-arm-model.js (hunk 1 above)
Comment (borismus) [reply src/orientation-arm-model.js]: Agree, please fix throughout.

File: src/ray-controller.js (hunk 4 above)
Comment (borismus) [reply src/ray-controller.js]: With parens is even better:

    this.isLeftHanded = (gamepad.hand === 'left');

**Review by borismus — COMMENTED** [review]
Thanks for this, @arturitu, and pardon long delay. Please update as per @cvan comments. In general we're sticking to [Google JS Style](https://google.github.io/styleguide/jsguide.html).

**Comment by arturitu** [comment]
I just fix those style issues, I hope all are ok. Thanks @cvan and @borismus for your comments.

File: src/orientation-arm-model.js [hunk 5]
```diff
@@ -78,8 +80,12 @@ export default class OrientationArmModel {
   }
 
   setLeftHanded(isLeftHanded) {
-    // TODO(smus): Implement me!
     this.isLeftHanded = isLeftHanded;
+    if (isLeftHanded){
```
Comment (borismus) [inline src/orientation-arm-model.js:84]: nit: space after )

File: src/ray-controller.js [hunk 6]
```diff
@@ -79,7 +80,11 @@ export default class RayController extends EventEmitter {
 
     var gamepad = this.getVRGamepad_();
 
+    this.gamepad = gamepad;
     if (gamepad) {
+      if(gamepad.hand){
```
Comment (borismus) [inline src/ray-controller.js:85]: plz fix style

**Review by borismus — COMMENTED** [review]
Couple more tweaks

**Pull request merged**